alea/submitters/run_toymc_wrapper.sh

```diff
@@ -93,6 +93,7 @@ METADATA=$(echo "$metadata" | sed "s/'/\"/g")
 
 # Installing customized packages
 . install.sh alea
```

Comment:
This can also be `. install.sh alea blueice`.

Reply:
done!